### ISSUE TITLE
join on empty dataframe returns an array instead of an empty dataset

### ISSUE THREAD

**Issue by fgutmann** [Gmousse/dataframe-js#54]
When joining with an empty dataframe the result is an array.
I'd expect it to be an empty dataframe.

Example:
```
let df1 = new dfjs.DataFrame([]);
let df2 = new dfjs.DataFrame([]);
df1.join(df2, 'col1'); // empty array
```

The empty array is quite unhandy in situations where `filter()` is followed by `join()`.
What are your thoughts on this?

**Comment by Gmousse** [comment]
Hum it's clearly a bug.
thanks @fgutmann I will see for a fix when I can find some free time.

**Comment by Gmousse** [comment]
It was fixed on this PR.

**Comment by fgutmann** [comment]
Thanks for being that fast 👍
A release would would be awesome too!

**Comment by Gmousse** [comment]
Fixed and published in 1.3.2
